### ISSUE TITLE
CDIA May 2023 Updates 

### ISSUE THREAD

**Issue by adamdecaf** [moov-io/metro2#187]
Placeholder issue for coordinating updates.

**Comment by dn768** [comment]
Hey Adam! Would you still like some help updating this project with the latest specifications?

I have the 2024 specs as well as the May 2024 list of revisions to work from if so.

Do you happen to know offhand if release v0.9.2 complies with a particular year of specification?

Thanks!

**Comment by adamdecaf** [comment]
I don't believe this has been updated for 2024 / 2025 support yet, we'd appreciate some help!
